{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "1.2.3.4, 1.2.3.4, 1.2.3.4, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}
{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "1.2.3.4, 1.2.3.4, 1.2.3.4, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}
{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "1.2.3.4, 1.2.3.4, 1.2.3.4, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}
{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "1.2.3.4, 1.2.3.4, 1.2.3.4, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}
{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "1.2.3.4, 1.2.3.4, 1.2.3.4, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}
{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "1.2.3.4, 1.2.3.4, 1.2.3.4, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}
